Subsurface/Node/tests/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.13)
+project(Node-Tests)
+
+set(CMAKE_CXX_STANDARD 17)
+
+include_directories(Unity/src)
+include_directories(../Common)
+include_directories(mock)
+
+add_compile_definitions(UNIT_TEST)
+
+set(HEADERS
+)
+
+set(SOURCES
+    Unity/src/Unity.c
+    ../Common/Peripherals/Src/can_manager.cpp
+    mock/mock_can_interface.cpp
```

Comment:
For the mock directory, what do you think we should either:
1. Having mock/inc and mock/src
2. Defining everything in one header for mock classes